app-backend/db/src/test/scala/com/azavea/rf/database/TaskDaoSpec.scala

```diff
@@ -1420,6 +1428,78 @@ class TaskDaoSpec
             )
             assert(randomTask3.isEmpty, "Task status filters are respected")
 
+            true
+          }
+      }
+    }
+  }
+  test("listing some children of a task") {
+    check {
+      forAll {
+        (
+            userCreate: User.Create,
+            orgCreate: Organization.Create,
+            platform: Platform,
+            projectCreate: Project.Create,
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            annotationProjectCreate: AnnotationProject.Create
+        ) =>
+          {
+            val connIO =
+              for {
+                (dbUser, _, _, dbProject) <- insertUserOrgPlatProject(
+                  userCreate,
+                  orgCreate,
+                  platform,
+                  projectCreate
+                )
+                dbAnnotationProj <- AnnotationProjectDao
+                  .insert(
+                    annotationProjectCreate.copy(
+                      projectId = Some(dbProject.id)
+                    ),
+                    dbUser
+                  )
+                tasks <- TaskDao.insertTasks(
+                  fixupTaskFeaturesCollection(
+                    taskFeaturesCreate,
+                    dbAnnotationProj
+                  ),
+                  dbUser
+                )
+                _ <- tasks.features.tail traverse { f =>
+                  TaskDao.updateTask(
+                    f.id,
+                    Task.TaskFeatureCreate(
+                      f.properties.toCreate
+                        .copy(parentTaskId = Some(tasks.features.head.id)),
+                      f.geometry,
+                      "Feature"
+                    ),
+                    dbUser
+                  )
+                }
+                children <- TaskDao.children(
+                  tasks.features.head.properties.id,
+                  PageRequest(0, 10, Map.empty)
+                )
+                parents <- TaskDao.listTasks(
```

Comment:
It's a little bit surprising to me that I can't deliberately filter for children -- is that a deliberate choice, or just an implementation convenience?